### DISCUSSION TITLE
QLMarkdown as Command-line tool

### DISCUSSION THREAD

**Discussion by setanarut** [sbarex/QLMarkdown#75] [Q&A] [answered]
Hi, I want to use QLMarkdown on command line to convert .md file to .html file by right click (with Automator workflow).  

for example;

```sh
originalFilePath=$1
uuid=$(uuidgen)
uuidFilePath=$originalFilePath-$uuid

/Applications/QLMarkdown.app/Contents/MacOS/QLMarkdown -i $originalFilePath -o $uuidFilePath.html
```

Is there a way I can use it as a converter, I need help.?

**Comment by sbarex** [comment]
I'll try to implement it, stay tuned.

**Comment by sbarex** [comment, marked as answer]
I implemented it in version 1.0.3. See the readme for usage instructions.

**Reply by setanarut** [reply]
Excellent thanks.
